### PR TITLE
[MAR-3239] Master Agreement

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html [hunk 1]
```diff
@@ -0,0 +1,487 @@
+<html>
+
+<head>
+
```
[the remark this answers is not in the document]
Comment (malmas18) [reply app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html]: added

File: app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+	<li>Adequately explain key terms and symbols.</li>
+	<li>Not contain any unusual omissions or exclusion.</li>
+	<li>Be in correct English.</li>
+</ul>
+
+<p><strong>8. Intellectual property rights</strong></p>
+
+<p><em>General</em></p>
+
+<p>You must ensure that the buyer&rsquo;s use of the order material will not infringe the intellectual property rights of any person.</p>
```
[the remark this answers is not in the document]
Comment (malmas18) [reply app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html]: updated

File: app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html [hunk 3]
```diff
@@ -0,0 +1,490 @@
+<html>
+
+<head>
+	<head>
```
Comment (tobiasbrown) [inline app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html]:
```suggestion
```

File: app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html [hunk 4]
```diff
@@ -0,0 +1,490 @@
+<html>
+
+<head>
+	<head>
+	<meta charset=utf-8>
+	<title>Master Agreement</title>
+	</head>
```
Comment (tobiasbrown) [inline app/static/media/documents/digital-marketplace-master-agreement-2019-07-01.html]:
```suggestion
```

**Pull request merged**